packages/components/src/form-token-field/stories/index.js

```diff
@@ -65,3 +68,35 @@ const FormTokenFieldAsyncExample = () => {
 export const _async = () => {
 	return <FormTokenFieldAsyncExample />;
 };
+
+const ComboboxExample = () => {
+	const [ selectedContinent, setSelectedContinent ] = useState( null );
+	const [ availableContinents, setAvailableContinents ] = useState( [] );
+	const searchContinents = ( input ) => {
+		const timeout = setTimeout( () => {
+			const available = continents.filter( ( continent ) =>
+				continent.toLowerCase().includes( input.toLowerCase() )
+			);
+			setAvailableContinents( available );
+		}, 1000 );
+
+		return () => clearTimeout( timeout );
+	};
+
+	return (
+		<>
+			<ComboboxControl
+				value={ selectedContinent }
+				suggestions={ availableContinents }
+				onChange={ ( tokens ) => setSelectedContinent( tokens ) }
+				onInputChange={ searchContinents }
+				placeholder="Type a continent"
```

Comment:
Translate?

Reply:
I believe it's fine here, it's just a story.